pywbem_os_setup.bat

```diff
@@ -36,6 +36,18 @@ goto end
 
 :do_install
 
+echo %myname%:
+echo %myname%: ================================================================
+echo %myname%:
+echo %myname%: Starting with pywbem 1.0.0, the installation of pywbem does not

```

Comment:
There may be some confusion over the word of installation since we really have two different installations:

1. user installation through pip

2. Development installation via clone.

We should clarify that.

Reply:
With item 2, you mean "make install"? That also uses pip, just with the work directory as a source.
In any case, the statement applies to both ways of installing pywbem.